specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/firewallPolicy.json

```diff
@@ -2053,11 +2071,13 @@
         "direction": {
           "type": "integer",
           "format": "int32",
-          "description": "Describes in which direction signature is being enforced: 0 - Inbound, 1 - OutBound, 2 - Bidirectional",
+          "description": "Describes in which direction signature is being enforced: 0 - OutBound, 1 - InBound, 2 - Any, 3 - Internal, 4 - InternalOutbound",
           "enum": [
             0,
             1,
-            2
+            2,
+            3,
+            4
```

Comment:
Why not use the string values OutBound, InBound etc.?

Reply:
What do you mean by shipped this definition?
The "direction" is in production and in use, but some of the values were missing here. We prefer not doing large changes, just fixing these values while we're at it :)